packages/dds/tree/src/core/tree/cursor.ts

```diff
@@ -9,11 +9,28 @@ import { FieldKey, TreeType, Value } from "./types";
 
 /**
  * A stateful low-level interface for reading tree data.
+ *
+ * @remarks Cursors exist so that specialized implementations can be written against different data abstractions.
```

Comment:
Cursor is the abstraction over different representations.

```suggestion
 * @remarks Cursor exists so that specialized data formats can be viewed through a common abstraction.
```